providers/class.two-factor-email.php

```diff
@@ -64,11 +64,11 @@ function authentication_page( $user ) {
 			}, 200);
 		</script>
 		<?php
-		submit_button( __( 'Log In', 'two-factor' ) );
+		submit_button( esc_html__( 'Log In', 'two-factor' ) );
 	}
 
 	function validate_authentication( $user ) {
-		return $this->validate_token( $user->ID, $_REQUEST['two-factor-email-code'] );
+		return $this->validate_token( $user->ID, $_POST['two-factor-email-code'] );
```

Comment:
This I had intentionally left as $_REQUEST in case we later modify the emails to include a link they can follow with the $_GET argument pre-filled.